pkg/actionsmetrics/event_reader.go

```diff
@@ -85,6 +88,9 @@ func (reader *EventReader) ProcessWorkflowJobEvent(ctx context.Context, event in
 			reader.Log.Info("reading workflow_job logs",
 				"job_name", *e.WorkflowJob.Name,
 				"job_id", fmt.Sprint(*e.WorkflowJob.ID),
+				"repository", *e.Repo.Name,
+				"repository_full_name", *e.Repo.FullName,
+				"organization", *e.Org.Name,
```

Comment:
I think this isn't safe because `Org` can be nil for repositories owned by a `User`. Let's add a nil check and add this label only when the workflow job is coming from a repo owned by an org!

Reply:
Updated